tests/integration/test_permissions_drop_replica/test.py

```diff
@@ -0,0 +1,64 @@
+import pytest
```

Comment:
I will also try to add a functional test for this.

Reply:
I tried working on a functional test but it looks like doing a full `SYSTEM DROP REPLICA` can be intrusive when run with other tests and in this particular case, I think that the integration test would be more reliable for this.